Userland/Libraries/LibWeb/Loader/LoadRequest.h

```diff
@@ -32,6 +34,9 @@ class LoadRequest {
     const ByteBuffer& body() const { return m_body; }
     void set_body(const ByteBuffer& body) { m_body = body; }
 
+    void start_timer() const { m_load_timer.start(); };
```

Comment:
This should be a non-const function.

Reply:
Sure, the const push down went pretty deep, but removing it wasn't that  bad.